runner/src/client/sass/application.scss

```diff
@@ -1,4 +1,4 @@
-@import "node_modules/govuk-frontend/govuk/all";
+@import "../../../../node_modules/govuk-frontend/govuk/all";
```

Comment:
if we nohoist govuk-frontend, it would mean it gets installed in [designer|runner]/node_modules <- but defeats the purpose/benefits of hoisting 😅. nvm!

Reply:
So annoying this